alumnos/vmf/clase7_ejercicios/5-6-7.regEx.js

```diff
@@ -0,0 +1,27 @@
+/*5. Crear una función que utilice una expresión regular para identificar si el parámetro str que recibe es una albóndiga. Tener en cuenta que (al igual que la RAE) hay que contemplar almóndiga. Debe ser case insensitive.*/
+
+function esAlbondiga(str) {
+    return /al(m|b)óndiga/i.test(str);
+}
+
+esAlbondiga(str);
+
+/*6. Crear una función que utilice una expresión regular para identificar si el parámetro str que recibe es la letra de la intro de la serie Batman de los 60. Dicha letra, de dificultad manificestat, dice lo siguiente: Na na na na na na na na na na na na na na na na... ¡BATMAN!. Debe ser case insensitive.*/
+
+function batmanSong(str) {
+    const itsdasong = /([n][a][ ]){15}... ¡BATMAN!/i;
```

Comment:
No necesitarías meter la n, la a y el espacio como grupo de caracteres! 
```suggestion
    const itsdasong = /(na ){15}... ¡BATMAN!/i;
```